typings/objection/index.d.ts

```diff
@@ -471,6 +471,7 @@ declare namespace Objection {
     // Non-query methods:
 
     context(queryContext: object): this;
+    context(): object;
```

Comment:
Isn't `object` as a return value pretty useless? You cannot access any property through it without first casting it to `any` or some other type. This should return `QueryContext` type.